packages/sequelize-transparent-cache/lib/cache/util.js

```diff
@@ -29,20 +29,31 @@ function dataToInstance (model, data) {
   return instance
 }
 
-function loadAssociations (model) {
+function loadAssociations (model, depth = 1) {
```

Comment:
This method is so mad, I have no idea how it works, please add tests to cover all code branches.

Reply:
After trying to add such test cases I figured those branches were unneeded as I always managed to get the correct model name on the first try. I went ahead and removed them